docker-compose.yaml

```diff
@@ -0,0 +1,43 @@
+version: "3"
+
+services:
+  politepol:
+    build:
+      context: .
+      args:
```

Comment:
Instead of the args, please add the same block as environment: args will use their defaults from the dockerfile, then the envs will let the user change them to whatever they like. Also, this way the toor pass will be leaked (who cares 😃) but not the final strong pass.